CI/markdownlint/style.rb

```diff
@@ -0,0 +1,8 @@
+all
+
+rule "no-duplicate-header", :allow_different_nesting => true
+rule 'line-length', :line_length=>100, :code_blocks=>false, :tables=> false
+rule 'no-trailing-punctuation', :punctuation=>'.,;:!'
```

Comment:
In `CONTRIBUTING.md` we have a header "Thanks for taking the time to contribute!", so perhaps remove "!" from the punctuation list.

Reply:
Will do in a content changing PR. This one is really only for the rules to be set.